ibm/resource_ibm_event_streams_topic.go

```diff
@@ -183,6 +182,7 @@ func resourceIBMEventStreamsTopicRead(d *schema.ResourceData, meta interface{})
 			return nil
 		}
 	}
+	d.SetId("")
```

Comment:
I think just setting the ID is not enough because even after setting the the ID here we are returing error and failing out. We should set the ID to empty and return successfully

Can you please test this scenario to validate this

1)create topic via terraform
2)Delete the topic manually uisng ui or ibmcloud cli
3)Run terraform plan (Here it shd plan out saying 1 new resource to be added)